### PR TITLE
Cluster trending task

### PR DESCRIPTION
**Pull request by ClaraBartels**
Online post-processing Cluster task for ITS

### CODE REVIEW

**Comment by iravasen** [comment]
PR reviewed on ITS side.

**Review by Barthelemy — APPROVED** [review]

**Comment by Barthelemy** [comment]
Ok for me but there is an issue with the formatting.

File: Modules/ITS/itsCluster.json [hunk 1]
```diff
@@ -20,11 +24,19 @@
         "url": ""
       },
       "conditionDB": {
+<<<<<<< HEAD
```
Comment (iravasen) [inline Modules/ITS/itsCluster.json:27]: Solve conflicts  in this file

**Review by ClaraBartels — COMMENTED** [review]
Resolved Conflicts

**Comment by iravasen** [comment]
@Barthelemy I don't see anymore the check on clang in the list and Clara applied it now. Can you try if it works? Thank you

**Comment by iravasen** [comment]
There is an issue with MonitorObject, Clara is solving it

**Pull request closed without merging**